Bottom: 87b979fc647e6cfacbbda727f84fabecde4e72a0
Top:    d89d2f10eb6998af6a91eff80f6a173702af6ef7
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-05-12 12:07:05 +0800

Refresh of tuntap-introduce-tx-skb-ring

---

diff --git a/drivers/net/tun.c b/drivers/net/tun.c
index e4c981a..8b6cf28 100644
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -2457,7 +2457,8 @@ static int tun_chr_open(struct inode *inode, struct file * file)
 	INIT_LIST_HEAD(&tfile->next);
 
 	sock_set_flag(&tfile->sk, SOCK_ZEROCOPY);
-	tfile->head = tfile->tail = 0;
+	tfile->head = 0;
+	tfile->tail = 0;
 
 	spin_lock_init(&tfile->rlock);
 	spin_lock_init(&tfile->wlock);
